demo/data-assimilation-4dvar/demo.ipynb

```diff
@@ -0,0 +1,855 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 4D-Variational Data Assimilation for a Chaotic Dynamical System\n",
+    "\n",
+    "In this tutorial, you will learn how to use differentiable Tesseracts to implement a 4D-Var data assimilation scheme for a chaotic dynamical system, which includes using Tesseracts in an optimization context and embedding them into other (differentiable) Python / JAX code. \n",
+    "\n",
+    "We will use the Lorenz 96 model as our dynamical system and demonstrate how to combine model forecasts with observations to produce an optimal estimate of the state of the system over a time window.\n",
```

Comment:
Add link/ref to Lorenz 96 maybe? Either to the wikipedia entry https://en.wikipedia.org/wiki/Lorenz_96_model or something better